src/lib/logger/vercel/utils.ts

```
@@ -0,0 +1,236 @@
+                    image_url: {
+                      url: msg.image.toString(),
+                    },
+                  };
+                default:
+                  return msg as any;
+              }
+            }),
+          }
+        ] as Array<CompletionRequest | ChatCompletionMessage>
+      case "assistant":
+        const assistantText = promptMsg.content.find((msg) => msg.type === "text");
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)